duckduckgo/macos-browser
# Remove DBP target and merge DBP to DevID target

Task/Issue URL: https://app.asana.com/0/1203581873609357/1205852422330137/f

**Description**:
Remove DBP target and merge it with non-sandbox one

**Steps to test this PR**:
(Make sure your locale is set to US and that you drag the dbp C-S-S package to the project)

1. Check if you are an internal user, you'll be able to apply for the waitlist
2. Check if you're NOT an internal user, you can't see the DBP menu item (even if your locale is US)
3. Run the non-sandbox target, check if you can start DBP and start scans
4. Run the privacy pro target, check if you can start DBP and start scans
5. Check if you can generate signed builds for non-sandbox target
6. Check if all tests are passing
